modules/preview-react/form-field/lib/hooks/useFormFieldModel.tsx

```diff
@@ -29,6 +29,12 @@ export const useFormFieldModel = createModelHook({
      * `required` set to true, and an asterisk will be appended to the `FormField.Label`.
      */
     isRequired: false,
+    /**
+     * True if the component and input should grow to its container's width. False otherwise.
+     * @default false
+     * @deprecated This property will be deprecated in a future major version. By default, inputs should be set to width 100% and the container should define its constraints.
+     */
+    grow: false,
```

Comment:
It seems weird to add a deprecated property.' `grow` just sets `width: '100%'`, right? Is `grow` really necessary?

Reply:
Well, no, but because our inputs don't default to width 100%, we need a way to add width 100% on the input as well as the container... Otherwise I'd have to add the cs prop with width on the container and the input... which is fine, just more tedious. Thoughts?